### DISCUSSION TITLE
Weekly update 06/22/2022

### DISCUSSION THREAD

**Discussion by dinaolmelak** [Swarmies/sap-core#45] [General]
# Dinaol

###  How do you feel today?

- Feeling good.   

### What did you do since the last meeting? 

- I had been working on my ticket. I had also created a PR for #43  and also made documentation edits for the ER_ Diagram documentation #44 .

### What are you planning to do moving forward? 

- Moving forward I'm going to use the schema I generated from lucid cart to create the databases. 

### Anything hampering your process? 

- Nothing is hampering my progress.

**Comment by Getnet12** [comment]
# Getnet

###  How do you feel today?

- I am feeling alright.   

### What did you do since the last meeting? 

- I have been playing around with docker containers. I have watched a couple of videos about react typescript in order to work with the student login page.

### What are you planning to do moving forward? 

- Moving forward, I'll work on the home, log in, and signup page of my ticket.

### Anything hampering your process? 

- a bunch of paper works for my job

**Comment by kirubeltadesse** [comment]
# Kirubel

###  How do you feel today?

- Overwhelmed 🧑‍💼 

### What did you do since the last meeting? 

- I was having a headache on Sunday afternoon. So I could do much but I tried to add a simple test case and configure the CI locally. 

### What are you planning to do moving forward? 

- I am planning to update PR #36 and get it merged. 

### Anything hampering your process? 
- Some paperwork I need to submit (SEVIS, etc) 
- Supper booked this week.
